pkgs/applications/audio/adlplug/default.nix

```
@@ -5,13 +5,13 @@
 
 stdenv.mkDerivation rec {
   name = "${pname}-${version}";
-  version = "v1.0.0-beta.5";
+  version = "v1.0.0";
```

```suggestion
  version = "1.0.0";
```
It's not typical to include a v prefix in the version string.